packages/team-management/src/components/TeamsAddEdit/index.tsx

```diff
@@ -237,10 +238,15 @@ export const TeamsAddEdit: React.FC<Props> = (props: Props) => {
 
           setPractitioners(filteredResponse);
         })
-        .catch(() => sendErrorNotification(lang.ERROR_OCCURRED));
+        .catch(() => {
+          setApiError(true);
+          sendErrorNotification(lang.ERROR_OCCURRED);
+        });
     }
   }, [disableTeamMemberReassignment, opensrpBaseURL, paginationSize, practitionersRole]);
 
+  if (apiError) return <BrokenPage />;
+
   if (!practitioners || (params.id && !initialValue)) return <Spin size={'large'} />;
```

Comment:
Does this also need to be refactored.

Reply:
i dont think so because initially their values are null and that is considored to be false and no matter even if an empty array is set from these values that be considered true so by logically this component should still work the same